pimpmycause/pimpmycause/templates/profiles/detail.html

```diff
@@ -7,10 +7,10 @@
     {% block main %}
     <section class="page-register pnl row">
         <div class="content-wrap">
-            <h1 class="pnl__title">Profile</h1>
+            <h1 class="pnl__title">{{profile_detail_form.name.value}}'s Profile</h1>
```

Comment:
h1 is missing from the page.
![image](https://cloud.githubusercontent.com/assets/13818956/24573769/e76b6280-167f-11e7-9fd1-f9d9b3cbddd2.png)